build.gradle

```diff
@@ -79,6 +82,21 @@ allprojects {
         configFile file("${project.rootDir}/config/checkstyle.xml")
     }
 
+    detekt {
+        config = files("${project.rootDir}/config/detekt/detekt.yml")
+        autoCorrect = false
+        buildUponDefaultConfig = false
+        disableDefaultRuleSets = false
+        ignoreFailures = false
+        parallel = false
```

Comment:
👋 @malinajirka !

From the documentation what this does is: `Builds the AST in parallel. Rules are always executed in parallel. Can lead to speedups in larger projects. `false` by default.`

AST, to the best of my knowledge, means `Abstract Syntax Tree`, which seems to speed up Detekt. I am not sure if that is the reason why this is not the default in general. I choose to have this `false` for two reasons:
1) It is the default, just to be on the safe side for now,
1) Running `Detekt` is already very fast, I don't think we need to worry in making it faster for our project. On the other hand, I worry that by executing rules in parallel can end up buggy and not report or report more violations than normal. Thus, I would start with `false`.

PS: We can definitely turn that on to `true`, I don't think it will hurt. But, the default `false` seems safer for me as a starting point.